assets/app/data/crowbar/services/upgrade-status.factory.spec.js

```diff
@@ -131,6 +136,77 @@ describe('Upgrade Status Factory', function () {
             should.exist(upgradeStatusFactory);
         });
 
+        describe('syncStatusFlags function', function () {
+            it('should be defined', function () {
+                should.exist(upgradeStatusFactory.syncStatusFlags);
+                expect(upgradeStatusFactory.syncStatusFlags).toEqual(jasmine.any(Function));
+            });
+
+            describe('when received status is completed', function () {
+                beforeEach(function () {
+                    bard.mockService(upgradeFactory, {
+                        getStatus: $q.when(completedUpgradeResponse)
```

Comment:
I'm not able to comment on line 6, but after talking with @MaximilianMeister its value should be `database` and not `admin_upgrade` (given the current status of the steps object).
I'm not sure how this would affect the rest of the tests.